### PR TITLE
Added the asteroid parameters of AsteroidImpact.

### PR DESCRIPTION
**Pull request by FlorisXll**
Added parameters of Asteroid Impact in the JSON resource file.

### CODE REVIEW

**Comment by dalderliesten** [comment]
@FlorisXll Tests failed.

File: src/main/resources/data/events.json [hunk 1]
```diff
@@ -25,6 +25,16 @@
     "description": "Get to work, the ship has been hit by an Asteroid!",
     "duration": 30000,
     "damage": 10
+    "parameters": [
+      {
+      	"key": "energyShield",
```
Comment (jessetilro) [inline src/main/resources/data/events.json]: This is a value, not a key. You should add a single parameter with key "location" and description something along the lines of "Where the malfunction is located".

**Comment by jessetilro** [comment]
LGTM

**Comment by dmalarev** [comment]
Looks good

**Pull request merged**